src/Widgets/NotificationEntry.vala

```diff
@@ -149,7 +175,27 @@ public class Notifications.NotificationEntry : Gtk.ListBoxRow {
         };
         revealer.add (deck);
 
-        add (revealer);
+        var overlay = new Gtk.Overlay ();
+        overlay.add (revealer);
+        overlay.add_overlay (delete_revealer);
+
+        var eventbox = new Gtk.EventBox ();
```

Comment:
This probably needs to be inside the deck. At the moment it appears to be blocking scroll events

Reply:
I'll move it :)